contracts/interfaces/IVault2.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title IVault
```

Comment:
can this be merged in IVault ?

Reply:
The only difference between the original `IVault` and this one is an additional parameter `address to` in `deposit` and `withdraw` methods, so I guess we can do something about it

Reply:
I would not add the address to
in this version of the Vault,

I know we through about this feature, but it's better to maintain the original vault interface at this point.